python_modules/dagster/dagster/core/definitions/graph.py

```diff
@@ -772,7 +779,7 @@ def _validate_out_mappings(
             if dynamic_handle and not mapping.definition.is_dynamic:
                 raise DagsterInvalidDefinitionError(
                     f'In {class_name} "{name}" output "{mapping.definition.name}" mapping from '
-                    f'solid "{mapping.maps_from.solid_name}" must be a DynamicOutputDefinition since it is '
+                    f"{target_solid.describe_node()} must be a DynamicOutputDefinition since it is "
```

Comment:
Does this need to change to `DynamicOut`? Doesn't need to be in this PR.

Reply:
Yup. I think that should be a follow up.